library/create_kafka.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/python
+
+
+from __future__ import (absolute_import, division, print_function)
+from sys import api_version
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: create_kafka
+
+short_description: Create Red Hat OpenShift Streams for Apache Kafka Instance
+
+# If this is part of a collection, you need to use semantic versioning,
+# i.e. the version is of the form "2.5.0" and not "2.4".
+version_added: "0.1.0"
+
+description: Create Red Hat OpenShift Streams for Apache Kafka Instance
+
+options:
+    name:
+        description: Name of the Kafka instance
+        required: true
+        type: str
+    cloud_provider:
+        description: Cloud provider for the Kafka instance
+        required: false
+        type: str
+        default: aws
+    region:
+        description: Region of the Kafka instance
+        required: false
+        type: str
+        default: eu-west-1
+    reauthentication_enabled:
```

Comment:
I think we need to have some form of prefix [advanced] [expert] for some flags for typical people
to stop reading at this point. All those flags would have been rarely used in some corner cases.

Reply:
this is a great shout. so from here down then we can recommend the default unless the user explicitly wants a different config?